RecoPixelVertexing/PixelTriplets/python/quadrupletseedmerging_cff.py

```diff
@@ -62,6 +62,8 @@
 )
 from Configuration.Eras.Modifier_trackingPhase1_cff import trackingPhase1
 trackingPhase1.toModify(PixelSeedMergerQuadruplets, **_forPhase1)
+from Configuration.Eras.Modifier_trackingPhase1QuadProp_cff import trackingPhase1QuadProp
+trackingPhase1QuadProp.toModify(PixelSeedMergerQuadruplets, **_forPhase1)
 from Configuration.Eras.Modifier_trackingPhase1PU70_cff import trackingPhase1PU70
 trackingPhase1PU70.toModify(PixelSeedMergerQuadruplets, **_forPhase1)
```

Comment:
I will remove `trackingPhase1PU70` as soon as we have pre2 and this PR merged (whichever happens last). The only reason I didn't include it here was to avoid possible merge conflicts with phase2 developments targeting pre2.